### ISSUE TITLE
A wrong path

### ISSUE THREAD

**Issue by ZcarecroW** [TomByrne/Flash2Svg#28]
Hello, I just installed the latest version of Flash2Svg (3.48.0) but when I try to export a very simple animation I get the following error:
![2016-11-12 19_26_05-adobe animate](https://cloud.githubusercontent.com/assets/5858880/20240042/07045856-a90f-11e6-8060-86cd08926523.png)
I'm from Germany and my Animate is German, so the text is saying:
_The given output path could not be found. Please update and try again. [D-\SOUwut\frizzy-ohr-wackel_temp\]_
The last must be _D:\SOUwut..._ but there is _ instead of : 
Please help. I like your plugin very much, used an earlier version before.

**Comment by ZcarecroW** [comment]
And there is the following error in the bottom field "Ausgabe" - I think it can be translated as "Output":
![2016-11-12 19_52_49-](https://cloud.githubusercontent.com/assets/5858880/20240148/ac927c56-a911-11e6-8902-370cb80e44d6.png)

**Comment by TomByrne** [comment]
Would you be able to attach the FLA?

**Comment by ZcarecroW** [comment]
Of course:
[frizzy-ohr-wackel.zip](https://github.com/TomByrne/Flash2Svg/files/588654/frizzy-ohr-wackel.zip)

**Comment by ZcarecroW** [comment]
btw: I also tried to install the extension manually like it is described here:
http://www.tbyrne.org/manually-installing-flash2svg
But the problem remained.

**Comment by mellipuh** [comment]
Any news here? I got the exact same problem (using Animate CC 2017)

**Comment by no-use-for-a-username** [comment]
Did anyone find a solution for this? I have the same problem (macOS 10.12.1, Adobe CC 2017).
![screen](https://cloud.githubusercontent.com/assets/24220317/20660600/cd848dba-b54a-11e6-87f0-dc728e853f21.png)

**Comment by mattlegaspi** [comment]
Same issue here 
![image](https://cloud.githubusercontent.com/assets/3173448/20933374/d90787a8-bba5-11e6-9641-c19f135c2d5b.png)

**Comment by VincentVToscano** [comment]
Same issue as others describe above.

1. OS X 10.11.6  
2. Animate CC 2017 Release Build 16.0.0.112  
  
The plugin creates the temp directory in correct location but nothing gets created. The path looks exactly as others have posted above (notice double slash [not sure if that was in previous versions]).

Output Window:
```TypeError: svgXml.defs[0] is undefined```

Nothing in the log file except below:
```Operation,Total Time,Average Time,Run Count```

Installed with:
```ZXPInstaller.app```

Let me know if you need more details.

**Comment by TomByrne** [comment]
Hi everyone,
I don't have time at the moment to look into this, hopefully within the next few weeks though.
The plugin creates a temporary movieclip filled with all of the shapes it needs.
Then it exports this using Animate's built-in SVG exporter.
Then it reads the generated SVG file and pulls out all of the shapes and puts them into the animated SVG file.

Looks as if the built-in SVG export is failing on Mac in v2017, not sure why though.

**Comment by alssl-ross** [comment]
Hi Tom,

Just to add, I had the extension installed and working and had produced several SVG animations. I hadn't updated Animate CC in some time though.

Today, I let Animate run through and auto-update and the extension now fails with the "the output directory could not be found" error. 

My Animate CC has been updated to 2017.0.1 release, Build 16.0.1.119 and I cannot export any FLAs to SVG now, even FLAs that I had successfully exported under an older version of Animate CC. This is running on Windows 10 Home.

Thanks for any help
